DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -141,8 +144,11 @@ const char* const TaggingVariableTokens[] = {
   /* [jetPt]                                    = */ "jetPt",
```

Comment:
May I suggest to enter at least some dummy values in TaggingVariableDescription
https://github.com/cms-sw/cmssw/pull/17247/files#diff-89cfcec978581c291c1e7822bb06e964R137

I don't care as much about the content of the description. The problem is that there are no values set,
which can lead to crashes if there is some code using descriptions for printouts or something.